test/test_other.py

```diff
@@ -13476,6 +13477,38 @@ def test_unistd_close_noderawfs(self):
     self.emcc_args += ['--pre-js', 'pre.js']
     self.do_run_in_out_file_test('unistd/close.c')
 
+  @requires_node
+  def test_noderawfs_override_standard_streams(self):
+    self.set_setting('NODERAWFS')
+    self.set_setting('FORCE_FILESYSTEM')
+    create_file('pre.js', '''
+let stdout = '';
+let stderr = '';
+
+Module['print'] = (text) => stdout += text;
+Module['printErr'] = (text) => stderr += text;
+Module['postRun'] = () => {
+    assert(stderr == '', 'stderr should be empty. \\n' +
+        'stderr: \\n' + stderr);
+    assert(stdout.startsWith('hello, world!'), 'stdout should start with the famous greeting. \\n' +
+        'stdout: \\n' + stdout);
+}
+''')
+    self.emcc_args += ['--pre-js', 'pre.js']
+    self.do_runf(test_file('hello_world.c'))
+
+  @requires_node
+  def test_noderawfs_override_stdin(self):
+    self.set_setting('NODERAWFS')
+    self.set_setting('FORCE_FILESYSTEM')
+    self.set_setting('EXIT_RUNTIME')
+    create_file('pre.js', '''
+const data = 'hello, world!\\n'.split('').map(c => c.charCodeAt(0));
+Module['stdin'] = () => data.shift() || null;
```

Comment:
It looks like we don't have any existing tests for `Module['stdin'`].   Can you perhaps at this new test here can call it `test_module_stdin`?  And then annotate it with `also_with_noderawfs` (and `@crossplatform`).

Reply:
Addressed with commit f927a004df016ad234b65e4f4b130f22a3044626.